.codesandbox/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:20-alpine
```

Comment:
looks like there is also [`node:lts-alpine`](https://github.com/docker-library/repo-info/blob/master/repos/node/remote/lts-alpine.md) available - we should use this instead, to avoid having to maintain the version here

Reply:
Changed  to `node:lts-alpine` 
[2fed6e3](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/294/commits/2fed6e355e191abf9c4ea9ccb18e7bff00648058)